documentation/src/main/asciidoc/repositories/Repositories.adoc

```diff
@@ -325,7 +326,7 @@ void add(Book... books);
 [NOTE]
 ====
 A future release of Jakarta Data might expand the list of built-in lifecycle annotations.
-In particular, we're hoping to add `@Persist`, `@Merge`, `@Refresh`, `@Lock`, and `@Remove`, mapping to the fundamental operations of `EntityManager`.
+In particular, we're hoping to add `@Persist`, `@Merge`, `@Refresh`, `@Lock`, and `@Remove` mapping to the fundamental operations of `EntityManager`.
```

Comment:
Revert, I think that comma is necessary, even if the sentence structure is admittedly a bit inelegant.